.travis.yml

```diff
@@ -1,5 +1,5 @@
 # make it explicit that we favor the new container-based travis workers
-sudo: false
+sudo: true
```

Comment:
Instead of using sudo, you should use the apt addon:

https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-with-the-APT-Addon